plugins/item_tasks/web_client/stylesheets/selectTaskWidget.styl

```diff
@@ -0,0 +1,16 @@
+.g-task-custom-target-container
+  margin-bottom 8px
+
+  .g-target-result
+    padding 5px 10px
+    background-color #eee
+    border-radius 3px
+    margin-bottom 6px
+
+.g-selected-task
```

Comment:
If this is always under `.g-task-custom-target-container`, let's indent it to be under that in this file.